### PR TITLE
feat: nft onboarding

### PR DESCRIPTION
**Pull request by agraebe**

### CODE REVIEW

File: src/pages/write-smart-contracts/tokens.md [hunk 1]
```diff
@@ -43,6 +43,28 @@ specifies the standard for fungible tokens on the Stacks blockchain. This specif
 that a fungible token on Stacks _must_ have. By complying with this standard, fungible tokens on Stacks can be easily
 represented by wallets that support Stacks.
 
+### Understanding the fungible token standard
+
+The [SIP-010][] standard is an interface definition that allows Stacks applications and wallets to interact with
+fungible tokens in a standard way. Supporting the standard reduces complexity for token creators to get their tokens
+into the ecosystem. Under the [SIP-009][] standard, fungible tokens must have the following characteristics:
```
Comment (jdbender66) [inline src/pages/write-smart-contracts/tokens.md:50]: I believe this link should go to SIP-010 instead of SIP-009?

**Comment by agraebe** [comment]
note: we should add docs for the new NFT API endpoint

**Review by pgray-hiro — CHANGES_REQUESTED** [review]
Some minor changes in the language used here. Otherwise looks good. I've tried to capture my recommendations into our budding style guide.

File: src/pages/write-smart-contracts/nft.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+
+## Step 6: Add tests
+
+At this point, the contract functions as intended, and can be deployed to the blockchain. However, it is good practice
+to write automated testing to ensure that the contract functions always perform in the expected way. When adding
+complexity or changing the contract, having pre-written, working tests can help you verify that changes you make don't
+alter the way that contract functions behave.
+
+Open the `tests/my-nft_test.ts` file in your IDE. In this step, you will add a single automated test to verify the
+``get-last-token-id` and `get-token-uri` functions of the contract.
```
Comment (agraebe) [inline src/pages/write-smart-contracts/nft.md:228]: one ` too much

File: src/pages/write-smart-contracts/nft.md [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+
+=> You have now learned how to work with contract traits, and how to unit test a contract with Clarinet. If you would
+like to try deploying your contract to the testnet, proceed with the following optional step.
+
+## Optional: Deploy the NFT to the testnet
+
+For this tutorial, you'll use the [testnet sandbox](https://explorer.stacks.co/sandbox/deploy?chain=testnet) to deploy
+your smart contract. Make sure you have connected your [Stacks web wallet](https://www.hiro.so/wallet/install-web) to
+the sandbox using the **Connect wallet** button, then copy and paste the `my-nft.clar` smart contract into the Clarity
+code editor on the **Write & Deploy** page. Edit the contract name or use the randomly generated name provided to you.
```
Comment (agraebe) [inline src/pages/write-smart-contracts/nft.md:269]: we can link to this view directly now

File: src/pages/write-smart-contracts/nft.md [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+
+Click **Deploy** to deploy the contract to the blockchain. This will display the Stacks web wallet window with
+information about the transaction. Verify that the transaction looks correct, and the network is set to `Testnet`, and
+click **Confirm**.
+
+The deployment process can take up to 15 minutes to complete. You can review it on the [transactions](https://explorer.stacks.co/transactions?chain=testnet)
+page of the explorer, or in the activity field of your web wallet.
+
+When your contract is confirmed, navigate to the **Call a contract** page of the sandbox, and search for your contract.
```
Comment (agraebe) [inline src/pages/write-smart-contracts/nft.md:278]: we can also link to this view directly

**Review by agraebe — COMMENTED** [review]
looks good to me! left a few minor suggestions

**Review by pgray-hiro — APPROVED** [review]

**Pull request closed without merging**